drracket/drracket/sprof.rkt

```diff
@@ -295,17 +292,16 @@
           (define show/hide-menu-item #f)
           
           (define/public (show/hide-sprof-panel show?)
-            (let ([main-children (send main-panel get-children)])
-              (send show/hide-menu-item 
-                    set-label
-                    (if show? sc-hide-sprof sc-show-sprof))
-              (unless (or (and show? (= 2 (length main-children)))
-                          (and (not show?) (= 1 (length main-children))))
-                (send main-panel change-children 
-                      (λ (l) 
-                        (if show?
-                            (list everything-else sprof-main-panel)
-                            (list everything-else)))))))
+            (define main-children (send main-panel get-children))
+            (send show/hide-menu-item set-label (if show? sc-hide-sprof sc-show-sprof))
+            (unless (or (and show? (= 2 (length main-children)))
+                        (and (not show?) (= 1 (length main-children))))
+              (send main-panel
+                    change-children
```

Comment:
Here too (`send` special treatment)

Reply:
Fixed in https://github.com/sorawee/fmt/pull/86